### ISSUE TITLE
Provide documentation on how to prevent connections that bypass dataplane and go directly into the service 

### ISSUE THREAD

**Issue by yskopets** [kumahq/kuma#335]
### Summary

Question from `Timotius` on [Slack](https://kuma-mesh.slack.com/archives/CN2GN4HE1/p1570515019073300):

> I've read the docs and starts the installation. Following https://kuma.io/docs/0.2.1/installation/ubuntu/ I map service to port 10000 to be accessed via data plane. But I still can access the service directly on port 9000. CMIIW, policies will not run if I access the service directly on port 9000. What is the good practice to avoid this?

**Comment by yskopets** [comment]
You’re absolutely right, if you access the service directly, policies won’t be applied.

There are several ways to prevent this:
1) if you’re using `Kuma` on `Kubernetes`, you get “transparent proxying” out-of-the-box. “transparent proxying” means that all incoming traffic will be automatically redirected into the `dataplane` (`envoy`). As a result, nobody will be able to access your service directly (except for containers collocated in the same `Pod`)
2) if you’re using `Kuma` in `universal` mode (e.g., on VMs), you have multiple options:
2.A. Set up “transparent proxying” similarly to `Kubernetes` case (technically speaking, it’s an IPtables configuration)
2.B. Bind your service to `loopback` interface (e.g., `127.0.0.1`). When your application is listening on `127.0.0.1:9000` instead of `0.0.0.0:9000`, it won’t be accessible from other hosts. However, other processes on the same VM will be able to access the app directly (by calling `127.0.0.1:9000`)
2.C. To completely isolate your service from other apps on the same VM, you need to put it into a separate `Linux network namespace` (that is what container environments, i.e. Docker, do for you automatically)

To sum up, if you’re using `Kuma` in `universal` mode, I would recommend you to start from configuration `2.B.`. If that is not enough, opt for `2.A.` or `2.C.`
